service/routes.py

```diff
@@ -43,15 +43,10 @@ def index():
 ######################################################################
 
 
-@api.route("/health")
-class HealthResource(Resource):
-    """ Healthcheck endpoints
-    GET /health - Returns status of the service
-    """
-
-    def get(self):
-        """ Let them know our heart is still beating """
-        return jsonify(status=200, message="Healthy")
+@app.route("/health")
```

Comment:
We shouldn't revert this. We should change the healthcheck path in the deployment.yaml instead